modules/teadsBidAdapter.js

```diff
@@ -127,6 +127,23 @@ function buildRequestObject(bid) {
   return reqObj;
 }
 
+function getSizes(bid) {
+  let sizes;
+  if (isVideoBid(bid)) {
+    sizes = utils.deepAccess(bid, 'mediaTypes.video.playerSize') ||
+      utils.deepAccess(bid, 'mediaTypes.video.sizes') ||
```

Comment:
I'm still not convinced that `mediaTypes.video.sizes` is a supported property :joy:

Reply:
PlayerSize is optional, bid.sizes is the default and as you we are not sure about `mediaTypes.video.sizes` (We didn't find any line code about this property in the sources) but there is an adUnit section in the Prebid documentation which makes us doubt (see my screen). That is the reason why we preferred to add this check anyway... 
<img width="901" alt="capture d ecran 2019-01-08 a 14 45 49" src="https://user-images.githubusercontent.com/8831565/50834517-7a353a00-1354-11e9-92af-82af3b1ecaee.png">

Reply:
Thanks for clarifying this. Even js is untyped, I pretend it is :wink: and try to avoid ambiguous variables. This fix was necessary as some internal behavior of `adunit.sizes` changed and the `playerSize` was not used anymore. That's why highlighted this in this PR.

When I searched the code for `sizes` related functions, I got really confused as there are multiple representations (as you mentioned single-size array and array of size arrays, but also the `<witdh>x<height>` arrays) and various parsing methods.

Hope this implementation will last :heart:  Thanks @valsouche for taking care :kissing_heart: